package/python-kiwi-spec-template

```diff
@@ -617,17 +618,13 @@ Provides manual pages to describe the kiwi commands
 %prep
 %setup -q -n kiwi-%{version}
 
-# Drop shebang for kiwi/xml_parse.py, as we don't intend to use it
-# as an independent script
-sed -e "s|#!/usr/bin/env python||" -i kiwi/xml_parse.py
-
 %build
 # Build C-Tools
 make CFLAGS="${RPM_OPT_FLAGS}" tools
 
 %install
-# Install Python 3 version
-python3 setup.py install --prefix=%{_prefix} --root=%{buildroot} %{?is_deb:--install-layout=deb}
+# Install Python 3 version from wheel
+python3 -m installer --destdir %{buildroot} %{SOURCE2}
```

Comment:
What we're missing here is a build step from source. In the Fedora spec, I have the following:

```
%build
# Because there are some compiled stuff
%set_build_flags

%py3_build

# Build man pages
make -C doc man

# Build C-Tools
make CFLAGS="%{build_cflags}" tools

%install
%py3_install

# Install C-Tools, man-pages, completion and kiwi default configuration (yes, the slash is needed!)
make buildroot=%{buildroot}/ install

# Install dracut modules (yes, the slash is needed!)
make buildroot=%{buildroot}/ install_dracut

[...]
```

Reply:
ok I see... more changes needed. Thanks

Reply:
Thanks I did so without using the macros as they differ between the distros